index.html

```diff
@@ -105,6 +105,7 @@ <h2 id="contribute">Contribute</h2>
 					<li><a style="font-weight:bold" href="https://github.com/git-for-windows/git/wiki/FAQ" target="_blank">FAQ</a></li>
 					<li><a href="https://github.com/git-for-windows/git" target="_blank">Repository</a></li>
 					<li><a href="http://groups.google.com/group/git-for-windows" target="_blank">Mailing List</a></li>
+					<li><a href="https://github.com/git-for-windows/git/releases/">All Releases</a></li>
```

Comment:
Maybe just call the link "Releases" or "Downloads" or even "Old Releases"? @dscho what do you think?